learning/tour-of-beam/learning-content/introduction/introduction-concepts/runner-concepts/description.md

```diff
@@ -61,9 +61,9 @@ In java, you need to set runner to `args` when you start the program.
 {{end}}
 
 {{if (eq .Sdk "python")}}
-In the Python SDK , the default is runner **DirectRunner**.
+In the Python SDK , the **DirectRunner** is the default runner and is used if no runner is specified.
```

Comment:
No action required.

Obligatory complaint that we never explain anywhere that the Direct Runner isn't a monolith and has very different behaviors between SDKs. I can't finish Prism soon enough...